packages/components/src/Checkbox/Checkbox.css

```diff
@@ -45,6 +50,15 @@
   border-color: var(--color-disabled--secondary);
 }
 
+:global(.jobber-retheme) .checkBox {
+  border-color: var(--color-border);
+  border-radius: calc(var(--radius-base) / 2);
```

Comment:
@chris-at-jobber should we do this? Or just set to radius-small and then bump small up to 4 and base up to 8?

Reply:
Yeah that's cleaner, I'll do that

Reply:
Just realized I also need to update the design package to add a radius-small but we need to do that anyways so why not now 😄